### DISCUSSION TITLE
Where is installed ImageMagick in Centos 8

### DISCUSSION THREAD

**Discussion by dtorvisco** [ImageMagick/ImageMagick#2825] [General]
I'm tried to use ImageMagick with Liferay 6.2 (Centos 8) and this CMS required to me the path where is installed. Where is it?

Thanks

**Comment by urban-warrior** [comment]
https://www.itzgeek.com/post/how-to-install-imagemagick-and-php-imagick-on-centos-8-rhel-8/
